### PR TITLE
Register prefix `expltools`

### PR DESCRIPTION
**Pull request by Witiko**
This PR registers the prefix ~~`expl3tools`~~ `expltools`.

### CODE REVIEW

**Comment by muzimuzhi** [comment]
It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/. Are you really going to change catcode of `3` locally and define things like `\l__expl3tools_..._tl` and `\expl3tools_...:n`?

**Comment by Witiko** [comment]
@muzimuzhi That's the plan.

File: l3kernel/doc/l3prefixes.csv [hunk 1]
```diff
@@ -78,6 +78,7 @@ etex,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https:
 etl,etl,Jonathan P. Spratte,https://github.com/Skillmon/ltx_etl,git@github.com:Skillmon/ltx_etl.git,https://github.com/Skillmon/ltx_etl/issues,2021-08-16,2021-08-16,
 exp,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2012-09-27,2012-09-27,
 expl,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2012-09-27,2012-09-27,
+expl3tools,expl3tools,Vít Starý Novotný,https://github.com/witiko/expl3tools/,https://github.com/witiko/expl3tools.git,https://github.com/witiko/expl3tools/issues,2024-06-26,2024-06-26,Development tools for expl3 programmers
```
Comment (Witiko) [inline l3kernel/doc/l3prefixes.csv:81]:
```suggestion
expltools,expltools,Vít Starý Novotný,https://github.com/witiko/expltools/,https://github.com/witiko/expltools.git,https://github.com/witiko/expltools/issues,2024-06-26,2024-06-26,Development tools for expl3 programmers
```

**Comment by Witiko** [comment]
If you'd like to keep the prefix reserved for internal use, please let me know and I will come up with a different one.

> It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/.

I am registering early, so that we can quibble about the prefix and potentially change it without a large impact on the code base.

**Comment by Witiko** [comment]
> It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/. Are you really going to change catcode of `3` locally and define things like `\l__expl3tools_..._tl` and `\expl3tools_...:n`?

@muzimuzhi: In commit d8d6536 from this PR, I changed the prefix from `expl3tools` to just `expltools`.
In the merged PR https://github.com/Witiko/expltools/pull/3, I changed the code, so that it uses the prefix `expltools`.

@josephwright: Are there any remaining concerns here?

**Review by muzimuzhi — APPROVED** [review]

**Comment by car222222** [comment]
No, nothing more to say.  Go for it! 
(But `explatypus` would be good too:-).

**Comment by Witiko** [comment]
> (But `explatypus` would be good too:-).

Explatypus may still make a comeback as one tool underneath the expltools umbrella. :wink:

**Comment by car222222** [comment]
🙃

**Pull request merged**